src/certificates/data/thunks.js

```diff
@@ -69,3 +97,26 @@ export function deleteCourseCertificate(courseId, certificateId) {
     }
   };
 }
+
+export function updateActivationCertificate(courseId, path, activationStatus) {
```

Comment:
```suggestion
export function updateCertificateStatus(courseId, path, activationStatus) {
```
and further. Activation is not clear in this case

Reply:
`updateCertificateActivationStatus` ?

Reply:
It will be more appropriate to call `updateCertificateActiveStatus` so that we control active status but not activation. Activation is when we expect some user to do something to enable functionality on some entity. Here we control the status straight away so it is just `updateCertificateStatus` or `updateCertificateActiveStatus`